### PR TITLE
Update styles of guides [re #6226]

### PR DESCRIPTION
**Pull request by lshrinivas**

### CODE REVIEW

**Comment by lshrinivas** [comment]
Note merge to feature branch

File: lib/presenter/Chart.js [hunk 1]
```diff
@@ -185,9 +185,8 @@ class Chart extends Events {
         }, this);
 
         if (this.guideLayer) {
-            if (!this._guideLayerEl) {
-                this._guideLayerEl = this.d3Svg.append('g').attr('class', 'guide_layer');
-            }
+            this.d3Svg.select('.guide_layer').remove();
+            this._guideLayerEl = this.d3Svg.append('g').attr('class', 'guide_layer');
```
Comment (lshrinivas) [inline lib/presenter/Chart.js:189]: Without this, when graph data updates, the guides show up below the axis text labels leading to ugliness

File: lib/presenter/Chart.js [hunk 2]
```diff
@@ -467,7 +466,7 @@ class Chart extends Events {
         var axisWidth = this.width - (this.margins.left + this.margins.right);
         this.d3Svg.append('text')
             .attr('x', this.margins.left + (axisWidth / 2))
-            .attr('y', (this.height - (this.margins.bottom / 2)) + 5) // +5 because dominant-baseline doesn't work in IE
+            .attr('y', (this.height - (this.margins.bottom / 2)) + 10) // +10 because dominant-baseline doesn't work in IE
```
Comment (lshrinivas) [inline lib/presenter/Chart.js:469]: Reduce some space wastage at the bottom of the graph

File: lib/views/YYComparisonGuide.js [hunk 3]
```diff
@@ -56,10 +55,7 @@ class YYComparisonGuide extends GuideLayer {
                 .append('g')
                 .attr('class', 'axis_guide x_guide--full_length');
 
-            var yLabelMargin = 4;
-            guideLayer.drawXLineGuide(xGuideSelection, xGuideEnterSelection, chart, {
-                labelY: height - margins.bottom - yLabelMargin
-            });
```
Comment (lshrinivas) [inline lib/views/YYComparisonGuide.js:62]: This hack is no longer needed since all UI graphs now have default Nugget styles (showing x axis labels etc.)

**Comment by choochburger** [comment]
Looks good 👍  Maybe squash these commits before merging?

**Comment by lshrinivas** [comment]
Will do!

**Pull request merged**